Frameworks/PSMTabBar/Styles/PSMSequelProTabStyle.m

```diff
@@ -117,19 +121,8 @@ - (void)setOrientation:(PSMTabBarOrientation)value
 #pragma mark -
 #pragma mark Add Tab Button
 
-- (NSImage *)addTabButtonImage
-{
-    return _addTabButtonImage;
-}
-
-- (NSImage *)addTabButtonPressedImage
-{
-    return _addTabButtonPressedImage;
-}
-
-- (NSImage *)addTabButtonRolloverImage
-{
-    return _addTabButtonRolloverImage;
+- (NSImage *)addTabButtonImage {
+    return [_addTabButtonImage imageWithOverlayColor:[self isInDarkMode] ? [NSColor whiteColor] : [NSColor blackColor]];
```

Comment:
Just to be sure, will this button be magically re-rendered when the dark/light mode setting is toggled?

Reply:
Yesss